src/scss/skin-modern/components/_subtitlesettingspanel.scss

```diff
@@ -0,0 +1,35 @@
+@import '../variables';
+@import '../mixins';
+
+.#{$prefix}-ui-subtitles-settings-panel {
+  @extend %ui-settings-panel;
+
+  right: 3em;
+  width: 7em;
+
+  // bottom arrow from http://www.cssarrowplease.com/
+  &::after {
```

Comment:
How about we remove that arrow completely and with that get rid of the need for correct placement? Looks childish anyway. What do you think?

Reply:
hm yes we can do it. I'm fine with that.